templates/part.board.mainView.php

```diff
@@ -28,7 +28,7 @@
 	</form>
 	</div>
 </div>
-<div id="board" class="scroll-container">
+<div id="board" class="scroll-container" ng-click="sidebar.show=false">
```

Comment:
@juliushaertl ui-sref doesn't work as expected. sidebar will never open.. It was my first try